include/s3select_functions.h

```diff
@@ -98,7 +98,7 @@ class s3select_functions
     {"#when-than#", s3select_func_En_t::WHEN_THAN},
     {"#case-when-else#", s3select_func_En_t::CASE_WHEN_ELSE},
     {"coalesce", s3select_func_En_t::COALESCE},
-    {"cast", s3select_func_En_t::CAST}
+    {"cast_to_string", s3select_func_En_t::CAST_TO_STRING}
```

Comment:
not align with naming convention , such as "int" and "float".
it should be "string".